components/ui/Badge.vue

```diff
@@ -49,33 +49,16 @@ import LockIcon from '~/assets/images/utils/lock.svg'
 import CalendarIcon from '~/assets/images/utils/calendar.svg'
 import CloseIcon from '~/assets/images/utils/check-circle.svg'
```

Comment:
Do you think you could migrate these to omorphia icons?

Reply:
All icons ?

Reply:
In this file yeah. Whenever I touch any file, I tend to also migrate stuff there to omorphia whenever I can

Reply:
I can look to do that ^^

Reply:
But it requires an omorphia PR (some icons are missing).

Reply:
Don't worry about it then!